### ISSUE TITLE
Const declarations are not supported in strict mode.

### ISSUE THREAD

**Issue by X-Tender** [lukechilds/when-dom-ready#12]
I get this error when run on iOS9 iPad.
Do you have an idea why babel didn't transpile your "const" to var?

my loaders cfg (webpack 1.x)

```
loaders : [
	{
		test: /\.js?$/,
		exclude: /node_modules(?!(\/|\\)when-dom-ready)/,
		loaders: [ 'babel' ],
	},
],
```

**Comment by lukechilds** [comment]
Thanks for notifying me.

Babel isn't transpiling let/const because I'm just using the arrow fn and array.includes transform plugins. I didn't realise older versions of Safari don't allow const in strict mode. I've just included the block scoping plugin.

**Comment by lukechilds** [comment]
Infact, I may aswell just use the es2015 preset. I've updated to the preset to hopefully prevent any further reduction in browser compatibility: https://github.com/lukechilds/when-dom-ready/commit/c887ea376ec96ddbb4d9c36cd7fd377e1719bc49

**Comment by lukechilds** [comment]
@X-Tender just published v1.2.12. Can you update and let me know if that resolves your issue?

**Comment by X-Tender** [comment]
Works perfectly. Thanks for the fast response.
